### PR TITLE
Prep 3.2.0 release

### PR DESCRIPTION
**Pull request by alinabuzachis**
##### SUMMARY

Run add_docs, generate changelog

##### ISSUE TYPE

- Feature Pull Request

### CODE REVIEW

**Review by markuman — APPROVED** [review]

**Review by marknet15 — APPROVED** [review]

File: changelogs/changelog.yaml [hunk 1]
```diff
@@ -706,3 +706,44 @@ releases:
     fragments:
     - bump_release_version.yml
     release_date: '2022-02-15'
+  3.2.0:
+    changes:
+      breaking_changes:
+      - aws_ssm - on_denied and on_missing now both default to error, for consistency
+        with both aws_secret and the base Lookup class (https://github.com/ansible-collections/amazon.aws/issues/617).
```
Comment (tremble) [inline changelogs/changelog.yaml:713]: Should we have backported a *breaking* change?

**Review by tremble — COMMENTED** [review]
A little concerned that we backported a breaking change, but otherwise LGTM

File: changelogs/changelog.yaml (hunk 1 above)
Comment (alinabuzachis) [reply changelogs/changelog.yaml:713]: I thought we could include breaking changes in 3.2.0. What do you suggest to do? Do we want to revert it and include in 4.0.0 directly?

Comment (markuman) [reply changelogs/changelog.yaml:713]: Uh yes. I think it must be reverted vom `stable-3`

https://github.com/ansible-collections/amazon.aws/pull/618#issuecomment-1034254967

Comment (alinabuzachis) [reply changelogs/changelog.yaml:713]: Can I do it in this PR or do we need a separate one?

Comment (alinabuzachis) [reply changelogs/changelog.yaml:713]: Sorry, missed completely that comment.

Comment (markuman) [reply changelogs/changelog.yaml:713]: If possible I would give it a try.

```
git reset --hard aeba19a # commit before that merge
git cherry-pick 7297e31 c06d77c 42295a7 d9d5d0d # cherry pick what comes after
```

and push with force? But I've no idea if that's more a punk way and will result in merge conflicts?

Comment (alinabuzachis) [reply changelogs/changelog.yaml:713]: Ok, thanks. let me try.

Comment (markuman) [reply changelogs/changelog.yaml:713]: ...plus rebuild docs and changelog.  
naybe there is also a smoother "rebase" way.....

Comment (alinabuzachis) [reply changelogs/changelog.yaml:713]: I rebuilt docs, but I have to hard reset and do it from the beginning. It happen in the past the same thing with the docs. I am re-doing right now. I only used `git revert commit_hash`

Comment (alinabuzachis) [reply changelogs/changelog.yaml]: I then need to update the version_added to 4.0.0 for #618

**Review by markuman — APPROVED** [review]

**Review by jillr — CHANGES_REQUESTED** [review]
The changes to plugins/lookup/aws_ssm.py and tests/unit/plugins/lookup/test_aws_ssm.py don't seem to have been successfully reverted (we shouldn't have python changes in a prep PR).

**Comment by alinabuzachis** [comment]
> The changes to plugins/lookup/aws_ssm.py and tests/unit/plugins/lookup/test_aws_ssm.py don't seem to have been successfully reverted (we shouldn't have python changes in a prep PR - I might just be misunderstanding what's happening in #764 though).

@jillr Do we want to revert the changes in this PR or should I re-open #764 and merge first that one and then rebase this one?

**Comment by alinabuzachis** [comment]
I re-opened this one and will do the revert directly here https://github.com/ansible-collections/amazon.aws/pull/764 instead of having it in the same prep PR.

**Comment by jillr** [comment]
Yep, let's sort out the overall status of the stable-3 branch first and then rebase this PR for docs and changelog only

**Review by markuman — APPROVED** [review]

**Review by tremble — APPROVED** [review]

**Comment by alinabuzachis** [comment]
regate

**Pull request merged**